lib/client-assets.php

```diff
@@ -109,9 +109,16 @@ function gutenberg_register_scripts_and_styles() {
 
 	// Editor Styles.
 	wp_register_style(
+		'gutenberg-mce-skin',
+		includes_url( 'js/tinymce/skins/lightgray/skin.min.css' ),
```

Comment:
Yep could be nice to able to import the `icons` explicitely for example (that's what I needed here).